nlp/notebooks/stac_wrapper/stac_config.cfg

```diff
@@ -0,0 +1,20 @@
+[PEPS]
+PEPS_url = https://peps.cnes.fr/resto/api/collections/
+# PEPS_collections = ["S1", "S2", "S2ST", "S3"]
+# PEPS_formats = ["json", "atom"]
+
+[COPERNICUS]
+COP_url = https://scihub.copernicus.eu/dhus/search?q=
+COP_user = None
+COP_pwd = None
+
+[PAVICCS]
+PAVICCS_url = https://pavics.ouranos.ca/twitcher/ows/proxy/catalog/wps
+
+[catalogs]
+daccs = https://hirondelle.crim.ca/stac
```

Comment:
This will be different for each "DACCS" instance that deploys `pavics-jupyter-images`. 

We can provide predefined catalogs, but they should use the same "key" names as in https://github.com/DACCS-Climate/Marble-node-registry/blob/main/node_registry.json to avoid confusion.